api/metrics/rent_burdened_test.py

```diff
@@ -0,0 +1,77 @@
+import sys
+sys.path.append("../")
+
+from api.metrics.community import CommunityMetrics
```

Comment:
Looks like your tests are failing because you need to import your new metric class here instead of the community metrics class.

Reply:
Thanks for finding this!